resources/compass/charts/provisioner/values.yaml

```diff
@@ -9,7 +9,14 @@ deployment:
 database:
   schemaFilePath: "/assets/database/provisioner.sql"
 
-hydraURL: "http://ory-hydra-public.kyma-system:4444/oauth2/token"
+gardener:
+  project: "" # Gardener project connected to SA
+  kubeconfigPath: "/gardener/kubeconfig/kubeconfig"
+  kubeconfig: "" # Base64 encoded Gardener SA key
+  secretName: "gardener-credentials"
+  manageSecrets: true
```

Comment:
As in the https://github.com/kyma-project/kyma/pull/7154, I'm thinking about the `manageSecrets: true` value. I know, previously the approach has been approved, but should we really have it turned on by default? It prevents installing Compass for anyone without providing secrets manually before installation. Could you please toggle it in all places, and create override for our internal pipeline?

Reply:
I think the installation would succeed without providing the gardener kubeconfig , but the generated secret would have empty data, making the provisioning though gardener functionality non-functioning. 
Anyways I'm okay with changing the default behavior to `false`, but then we need to guard all the references to the secrets with `{{- if }} ` to avoid K8S throwing errors due to nonexistent Secret resources.

Reply:
I think by default the KEB and provisioner subcharts are guarded by the global.provisioning.enabled value, which is false by default. If someone enables it, the default behavior now is to supply credentials for gardener.

Reply:
I mean I think the secrets should be created by default.

I've thought incorrectly that `manageSecrets: true` means that the secrets are not created - but they are. Which is totally fine. Sorry for making confusion.